rest-api-model-server/src/main/kotlin/org/modelix/sample/restapimodelserver/Serialization.kt

```diff
@@ -1,35 +1,38 @@
 package org.modelix.sample.restapimodelserver
 
-import University.Schedule.structure.Courses
-import University.Schedule.structure.Rooms
-import org.modelix.model.lazy.INodeReferenceSerializer
+import University.Schedule.N_Courses
+import University.Schedule.N_Lecture
+import University.Schedule.N_Room
+import University.Schedule.N_Rooms
+import org.modelix.model.api.serialize
+
 
 /**
  * A Kotlin extension function to convert a model lecture to its JSON representation enforced by the generated
  * data class [Lecture].
  */
-fun University.Schedule.structure.Lecture.toJson() = Lecture(
-    lectureRef = INodeReferenceSerializer.serialize(this.reference),
-    name = this.properties.name ?: "",
-    description = this.properties.description ?: "",
-    maxParticipants = this.properties.maxParticipants ?: 0,
-    room = INodeReferenceSerializer.serialize(this.references.room.reference),
+fun N_Lecture.toJson() = Lecture(
+        lectureRef = this.unwrap().reference.serialize(),
+        name = this.name!!,
+        description = this.description!!,
+        maxParticipants = this.maxParticipants,
+        room = this.room!!.unwrap().reference.serialize(),
```

Comment:
It's not a good idea here to use `!!` as while editing in MPS, things like the description are still `null`. That's the reason for the implied defaults in the original version.